README.md

```diff
@@ -1,2 +1,14 @@
 # hw01
-Repository for homework 1
+Homework 1 from Donghyun Kang
+
+<img src="https://raw.githubusercontent.com/nerdizzyz/hw01/master/KANG_DONGHYUN.jpg">
```

Comment:
I was having trouble adding my own image  and ended up uploading it as a separate file. I didn't know about the "img src" function, so thank you! I will use it next time.